dialogue-core/src/main/java/com/palantir/dialogue/core/AimdConcurrencyLimiter.java

```diff
@@ -37,13 +39,13 @@
 final class AimdConcurrencyLimiter {
```

Comment:
This is not really additive increase multiplicative decrease is it... CautiousUpAggressiveDown?